oracle/oracles/pd.go

```diff
@@ -52,13 +53,16 @@ import (
 var _ oracle.Oracle = &pdOracle{}
 
 const slowDist = 30 * time.Millisecond
+const minTimestampUpdateInterval = 50 * time.Millisecond
+const maxTimestampUpdateInterval = 5 * time.Second
```

Comment:
If the configuration is to be used as an system variable, the validation could be done in the tidb side.

Reply:
It will be a system variable and will have its own validation there, but the client should probably have its own validation to make sure it doesn't get into a bad state. For example passing in duration of 0 will cause it to panic without any checking of its own.

Reply:
After thinking about this more I just made the validation check it won't panic and will let tidb do the additional validation as you suggested